base/logger/NEWS.md

```diff
@@ -0,0 +1,7 @@
+# PEcAn.logger 1.7.1.9000
+
```

Comment:
(Recording the change in behavior produced by my `dump.frames`->`sys.calls` change below -- only accept this if accepting the other change too!)

```suggestion
## Fixed

* Logger calls no longer create a stray `dump.log` object in the global environment
```